### ISSUE TITLE
martinezluciano | Lesson 1

### ISSUE THREAD

**Issue by martin5211** [martin5211/learn-node-js#3]
https://github.com/martin5211/learn-node-js/blob/2438157348d4c8d782ed2b3a9c5a8c82fee896cb/lesson_1/ejercicio/app.js#L7

- Would be needed to change the file format to JSON instead of html to comply with the exercise of Lesson 2. The non breaking spaces (`&nbsp`) shouldn't be added at all into a log file.

- You can use `process.argv` instead of yargs module. (https://nodejs.org/docs/latest/api/process.html#process_process_argv). I don't see `yargs` module used at all. The exercise requests the message to be passed by console argument.

- It's better to separate the writer functionality into a module called `logger` so it can be used in any part of the code.

- The `config.json` should be used to put the application configuration, like the name of the file to save.

- Avoid unnecessary blank lines at the end of the file.

**Comment by martinezluciano** [comment]
- have changed the file format log to .json. Also I changed the code to be consistent with that. The breaking spaces `&nbsp` have been deleted.
- I already added the `process.argv` with the functionality of receiving msg by console. 
- I already created the` logger.js` with the writer functionality. 
- The name (log.json) was taken from` config.json`, but unnecessary content was deleted from the file.
- Unnecessary blank lines were already deleted.
